content/en/infrastructure/containers/configuration.md

```diff
@@ -76,6 +76,121 @@ password::::== <MY_PASSWORD>
 
 However, the scrubber does not scrub paths that contain sensitive words. For example, it does not overwrite `/etc/vaultd/secret/haproxy-crt.pem` with `/etc/vaultd/******/haproxy-crt.pem` even though `secret` is a sensitive word.
 
+## Configure Container Images View
+
+### Image Collection
+
+Datadog collects container image metadata to provide enhanced debugging context. 
+
+#### Agent
+
+{{< tabs >}}
+{{% tab "Kubernetes (Helm)" %}}
+
+If you are using helm version `>= 3.46.0`, image collection is [enabled by default][3].</br>
+Or, add the following to your `values.yaml` helm configuration file:
+
+```yaml
+datadog:
+  containerImageCollection:
+    enabled: true
+  sbom:
+    containerImage:
+      enabled: true
+```
+[1]: https://github.com/DataDog/helm-charts/blob/main/charts/datadog/values.yaml#L651
+{{% /tab %}}
+
+{{% tab "Kubernetes (Operator)" %}}
+
+Add the following to the spec section of your `values.yaml` file:
+
+```yaml
+apiVersion: datadoghq.com/v2alpha1
+kind: DatadogAgent
+metadata:
+  name: datadog
+spec:
+  features:
+    # ...
+    sbom:
+      enabled: true
+      containerImage:
+        enabled: true
+```
+
+{{% /tab %}}
+
+{{% tab "ECS EC2" %}}
+
+To enable container image vulnerability scanning on your [ECS EC2 instances][4], add the following environment variables to your `datadog-agent` container definition:
```

Comment:
Link is broken

Reply:
This link is still broken. Please fix it by adding the link target **inside** the tab.